src/Nytte.Modules.Requests/ModuleRequestSpecification.cs

```diff
@@ -0,0 +1,21 @@
+using System;
+using Nytte.Modules.Requests.Abstractions;
+
+namespace Nytte.Modules.Requests
+{
+    public class ModuleRequestSpecification : IModuleRequestSpecification
```

Comment:
Again maybe change query to request. Only because query isn't used anywhere else.